db/migrate/20160415121724_create_cloud_services.rb

```diff
@@ -0,0 +1,21 @@
+class CreateCloudServices < ActiveRecord::Migration[5.0]
+  def change
+    create_table :cloud_services do |t|
+      t.string :ems_ref
+      t.string :source
+      t.string :binary
```

Comment:
Expanations are [here for source](https://github.com/ManageIQ/manageiq/pull/7996#discussion-diff-60134994) and [here for binary](https://github.com/ManageIQ/manageiq/pull/7996#discussion-diff-60135015).  I have a comment on `binary` below in the PR thread